pkg/context/blockchain/client.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package blockchain
```

Comment:
nitpick: did the reference implementation / sidetee spec use the term blockchain?

Reply:
yes